### PR TITLE
docs/getting-started: Tectonic for quick installs.

### PR DESCRIPTION
**Pull request by ElijahCaine**
Plugging our fancy tool.

@joshix @philips

### CODE REVIEW

File: Documentation/getting-started.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # CoreOS &#43; Kubernetes Step By Step
 
-This guide will walk you through a deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. We're going to:
+This guide will walks through the deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. This is a long form "hard way" guide that can be error prone but can help you understand how the system works. If you want to get started faster checkout [Tectonic Free Tier][tectonic-free] and the [Tectonic Installer][tectonic-installer].
```
Comment (robszumski) [inline Documentation/getting-started.md]: `check out`

File: Documentation/getting-started.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # CoreOS &#43; Kubernetes Step By Step
 
-This guide will walk you through a deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. We're going to:
+This guide will walks through the deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. This is a long form "hard way" guide that can be error prone but can help you understand how the system works. If you want to get started faster check out [Tectonic Free Tier][tectonic-free] and the [Tectonic Installer][tectonic-installer].
```
Comment (joshix) [inline Documentation/getting-started.md]: first sentence: "This guide walks through deploying a Kubernetes cluster of CoreOS nodes, with a single controller and multiple workers."
second sentence: "This guide enumerates the multiple steps and stages of a Kubernetes deployment."
third sentence: "To quickly deploy a Kubernetes cluster without engaging component-level details, check out the [free tier of the CoreOS Tectonic][tectonic-free] Kubernetes distribution, or the [open-source Tectonic Installer][tectonic-installer] that drives Tectonic's automation of cluster deployments."

File: Documentation/getting-started.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+
+More specifically, in this guide we will:
```
Comment (joshix) [inline Documentation/getting-started.md]: The two primary goals of this guide are to:

File: Documentation/getting-started.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
 
 - configure an etcd cluster for Kubernetes to use
```
Comment (joshix) [inline Documentation/getting-started.md]: Capital C Configure

File: Documentation/getting-started.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
...
 - generate the required certificates for communication between Kubernetes components
```
Comment (joshix) [inline Documentation/getting-started.md]: capital G Generate

**Comment by ElijahCaine** [comment]
I cannot merge this, FYI.

**Pull request merged**